demos/fab.html

```diff
@@ -39,12 +43,57 @@
           right: 5rem;
         }
       }
+
+      fieldset legend {
+        display: block;
+        padding: 16px;
+        padding-top: 64px;
+        padding-bottom: 24px;
+      }
+
+      .hero button {
+        margin-left: 32px;
+        margin-right: 32px;
+
+      }
+
+      /* Override colors to match catalog theme. */
+
+      .mdc-fab {
+        background-color: #212121;
+      }
+
+      .mdc-fab--plain {
+        background-color: #ffffff;
+      }
+
+
+      /* Override mini-FABs with smaller icon size. */
+      .mdc-fab--mini.material-icons .mdc-fab__icon {
+        font-size: 18px;
```

Comment:
The current icon is too big though for the mini fab. I'll take it up with the design team and remove it.